recipes/nmos-cpp/all/conanfile.py

```diff
@@ -56,11 +45,12 @@ def config_options(self):
 
     def requirements(self):
         # for now, consistent with project's conanfile.txt
-        self.requires("boost/1.79.0")
+        self.requires("boost/1.80.0")
         self.requires("cpprestsdk/2.10.18")
         self.requires("websocketpp/0.8.2")
-        self.requires("openssl/1.1.1o")
+        self.requires("openssl/1.1.1s")
         self.requires("json-schema-validator/2.1.0")
+        self.requires("zlib/1.2.13")
```

Comment:
```suggestion
        self.requires("zlib/1.2.13", override=True)
```

Reply:
override is forbidden on CCI. If some dependency requires a newer version, a new PR should be opened.